WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -745,6 +745,9 @@ public void onPause() {
     @Override
     public void onResume() {
         super.onResume();
+        if (!isAdded() || getView() == null) {
```

Comment:
I don't think this will help though, it looks like the line that's crashing is [line 2322 here](https://github.com/wordpress-mobile/WordPress-Android/blob/3f9aeaa9f7951a4740b710f44d4f4833f763e86c/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java#L2322), according to the stack trace of the crash:
```
java.lang.IllegalStateException: Fragment ReaderPostListFragment{b2c45df} (12cf4b19-e069-4888-ab5a-911d5ed99a51) not attached to an activity.
    at androidx.fragment.app.Fragment.requireActivity(Fragment.java:1000)
    at org.wordpress.android.ui.reader.ReaderPostListFragment$11.run(ReaderPostListFragment.java:2322)
```

If you look a couple of lines above the crashing line you will see that a similar check is already being made and even then the crash happened to a few users. I guess that, since the code is being run in another Thread, it might have been delayed enough that when line 2322 is reached the Fragment is no longer attached.

If that's indeed the case, we probably shouldn't be using `requireActivity` there, but instead just get the activity and only run the code **if** it is not null, since being null should mean that Fragment is no longer attached so that update is no longer needed anyway. Wdyt?

Reply:
Thank you for your suggestions. I've made the adjustments as recommended, although it resulted in about 4 isAdded checks and 2 getActivity() != null conditions within a few lines of code. 😵‍💫 

Side note: I believe the bottleneck isn't the `shouldAutoUpdateTag` function itself, but rather the delay in starting the thread, likely due to many other threads running concurrently to update the Reader in the background.

Regardless, we should have significantly reduced the likelihood of the crash occurring now.